### PR TITLE
[Feature] : Get in touch component

### PR DESCRIPTION
**Pull request by SahilVermaFED**
## Description

-   [Ticket Link]
-  Get in touch component afterwards will attach to footer section while making homepage

- ## UI screenshot or Video
[get-in-touch-component.webm](https://github.com/primathonacademy/april-batch-live-project/assets/131689244/b3159f62-24d6-4957-a152-90a6d963a3a9)

### General:

-   [x] **PR Size :** Confirming PR size < 20 files or pre-approved if > 20 files.
-   [x] **Self Review :** Confirming I have self-reviewed my PR
-   [x] **Peers Review :** - [ ] @Dev 1
-   [x] Confirming I have added all sensitive information-related should be inside .env files

### Client:

-   [x] **Console :** Confirming no console errors in the UI for this update.

-   [x] **Import order :** Confirming correct import order.

```
# External Import
import React, { useState, useEffect } from 'react';
import { useParams } from 'react-router-dom';
import { useDispatch, useSelector, shallowEqual } from 'react-redux';

# Internal Import
import { } from '../../Api/ApiRoutes';
import { } from '../../constants/AppConstant';
```

-   [x] **Re-renders :** Confirming minimal re-renders for new React components.

-   [x] **Network calls :** Confirming efficient network/API calls, minimized as needed.

-   [x] **Util function comments :** Confirming comprehensive comments for utility functions.

-   [x] **Image optimization :** Confirming I have checked that all images are optimized. If not, please use https://tinyjpg.com/ or https://vecta.io/nano.

-   [x] **Dead code :** Confirming I have removed all comments code which is of no use OR written why I have kept the commented code.

### Confirming different viewport testing of the entire flow:

-   [x] Tested on 280px viewport
-   [x] Tested on 320px viewport
-   [x] Tested on 375px viewport
-   [x] Tested on 425px viewport
-   [x] Tested on 768px viewport
-   [x] Tested on 1024px viewport
-   [x] Tested on 1440px viewport
-   [x] Tested on 2560px viewport

### End-to-end testing:

-   [x] **Happy flow :** Successfully tested end-to-end happy flow, all working as expected.
-   [x] **Error flow :** Tested UI flow with error scenarios, gracefully handled errors on UI
-   [x] **Common component changed :** Tested change in common component across views to prevent breaking elsewhere.

### CODE REVIEW

**Comment by ram-primathon** [comment]
its not matching the design please fix this

File: src/pages/homepage/GetInTouch.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { NavLink } from 'react-router-dom';
+import rightArrow from '../../../public/icons/right-arrow.svg';
```
Comment (ram-primathon) [inline src/pages/homepage/GetInTouch.jsx]: move this inside assets

File: src/pages/homepage/index.jsx [hunk 2]
```diff
@@ -1,5 +1,11 @@
+import GetInTouch from './GetInTouch';
+
 const Homepage = () => {
-    return <div className="px-6 md:px-16 lg:px-20 xl:px-28 2xl:px-36">Welcome to Homepage</div>;
+    return (
+        <div>
```
Comment (ram-primathon) [inline src/pages/homepage/index.jsx]: remove wrapper div

**Review by ram-primathon — CHANGES_REQUESTED** [review]

**Comment by TheSahilVerma** [comment]
> > its not matching the design please fix this
> 
> > when line deisgn is ready then i will merge all have the same svg problem thats why i didnt update bg-line deisgn

File: src/pages/homepage/GetInTouch.jsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import { NavLink } from 'react-router-dom';
+import rightArrow from '../../assets/icons/right-arrow.svg';
```
Comment (ram-primathon) [inline src/pages/homepage/GetInTouch.jsx]: this is wrong

**Review by ram-primathon — CHANGES_REQUESTED** [review]

**Review by ram-primathon — APPROVED** [review]

**Pull request merged**